### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by vishnupc**

### CODE REVIEW

**Review by vishnupc — COMMENTED** [review]
Added necessary details.

File: 3. Installation/2. PaaS Deployments/ReadySpace/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Deploying Rocket.Chat on ReadySpce Cloud
```
Comment (geekgonecrazy) [inline 3. Installation/2. PaaS Deployments/ReadySpace/README.md]: Looks like missing an `a` in Readyspace :smile:

**Review by geekgonecrazy — CHANGES_REQUESTED** [review]

**Comment by rafaelks** [comment]
@geekgonecrazy Can you review it again, please?

**Review by geekgonecrazy — APPROVED** [review]

**Comment by geekgonecrazy** [comment]
@vishnupc thanks for the contribution :+1:

**Comment by vishnupc** [comment]
Thank You @geekgonecrazy for merging my request.

**Pull request merged**